test/e2e/scalability/density.go

```diff
@@ -270,6 +270,8 @@ func runDensityTest(dtc DensityTestConfig, testPhaseDurations *timer.TestPhaseTi
 		}()
 	}
 	logStopCh := make(chan struct{})
+	go framework.GatherApiserverCPUProfile(&wg, "DENSITY_creating_saturation_pods")
```

Comment:
I'm not convinced how useful is that - this is really running on the empty cluster at the beginning of the test.

Reply:
Yeah.. that sounds reasonable. I replaced it with periodic polling now.